python/ray/actor.py

```diff
@@ -720,6 +721,34 @@ def _remote(self, args=None, kwargs=None, **actor_options):
         max_task_retries = actor_options["max_task_retries"]
         max_pending_calls = actor_options["max_pending_calls"]
 
+        if scheduling_strategy is None or not isinstance(
+            scheduling_strategy, PlacementGroupSchedulingStrategy
+        ):
+            if placement_group not in ("default", None):
+                warnings.warn(
+                    "placement_group parameter is deprecated. Use scheduling_strategy "
```

Comment:
Using `scheduling_strategy=PlacementGroupSchedulingStrategy(...)` instead, see the usage at...

here and below

Reply:
Good point, that's more informative!